src/components/PatronBlock/PatronBlockModalWithOverrideModal.test.js

```diff
@@ -0,0 +1,56 @@
+import { fireEvent, screen } from '@testing-library/react';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+import PatronBlockModalWithOverrideModal from './PatronBlockModalWithOverrideModal';
+
+import '__mock__/stripesCore.mock';
+import '__mock__/stripesSmartComponent.mock';
+
+jest.unmock('@folio/stripes/components');
+
+const renderPatronBlockModalWithOverrideModal = (props) => renderWithRouter(<PatronBlockModalWithOverrideModal {...props} />);
+
+describe('UserPermissions component', () => {
+  it('Checking patronBlock Modal', () => {
+    const props = {
```

Comment:
@Mathansivavelmurugesan it looks like the same structure is repeated in other tests. Would it be possible for you to move it maybe maybe to `beforeEach` and just reuse it between tests?